libp2p/Host.cpp

```diff
@@ -839,6 +817,12 @@ void Host::disconnectLatePeers()
 
 bytes Host::saveNetwork() const
 {
+    if (haveNetwork())
+    {
+        cwarn << "Cannot save network configuration since network is still running.";
+        return RLPStream{}.out();
```

Comment:
Just `bytes{}` would be simpler; looking at this I thought it might return empty RLP-list or something